### PR TITLE
Portfolio Accessibility

### PR DESCRIPTION
**Pull request by Simiran777**
### **WHAT ?**
1. The text alternatives for images were changed to be made detailed.
2. Labels were added for the Contact form fields.

### **WHY ?**
These changes were made in order to make the portfolio page accessible.

### **Following checks were undertaken-**
<hr>

**Page Titles** - No issues found

 **Image text alternatives** - The text alternatives for images were changed to be made detailed.

**Text Headings** - No issues found

**Color Contrast** - No issues found

 **Resize** - No issues found

 **Interaction** - Labels were added to the Contact form fields.

**Moving Content** - No issues found

**Multimedia** - No issues found

**The basic structure of the page** - No issues found

### CODE REVIEW

**Review by cynthiainga — COMMENTED** [review]
Hi @Simiran987, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn’t have a contribution from your partner, please this is the pair programming activity kindly make sure your partner contributes to this project, and if you have some issues whit your partner please contact the student success 🙃. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -37,11 +37,11 @@ <h1 class="intro">I’m Simiran <br> Glad to see you!</h1>
                 <div class="body2">
```
Comment (1GORDON) [inline index.html:37]: Great job @Simiran987 🤝

- [ ] Kindly make sure the text doesn't disappear or get cut off on the screen resize.
- [ ] Horizontal scrolling is not required to read sentences or "blocks of text". It is the best practice that when text size is increased, all the text in a sentence is visible.
resize

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Simiran987 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Simiran987         :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**